modules/repository/check.go

```diff
@@ -120,7 +120,7 @@ func gatherMissingRepoRecords(ctx context.Context) ([]*models.Repository, error)
 			return nil
 		},
 	); err != nil {
-		if strings.HasPrefix("Aborted gathering missing repo", err.Error()) {
+		if strings.HasPrefix(err.Error(), "Aborted gathering missing repo") {
```

Comment:
Pinging @zeripath since I don't know if that was an intentional change in #10745

Reply:
for me this looks like a bug ...
Definiton:
```go
func HasPrefix(s, prefix string) bool
```

Reply:
I think that was a bug I must have added! Good catch!